packages/website/src/demo-building-blocs/Sandbox.tsx

```diff
@@ -74,6 +74,13 @@ const Editor: React.FC<{id: string; value: string; onChange: (newValue: string)
     let editor: monaco.editor.IStandaloneCodeEditor;
     let model: monaco.editor.IModel;
     const onChangeEditor = React.useMemo(() => _.debounce(() => onChange(editor.getValue()), 500), []);
+
+    const updateHeight = () => {
+        const contentHeight = Math.min(1000, editor.getContentHeight());
+        divEl.current.style.height = `${contentHeight + 5}px`;
```

Comment:
Would it work with a min-height?

I'm wondering if in some cases the "demo" could be bigger than the code

Reply:
If you look on line 79, I have set a max of 1000 px

Reply:
Added a `min-height` of `100%` so that the editor minimally has the same height as the preview.
Also bounded the expansion and shrinking of the editor between 150 and 600 pixels.